packages/web/src/providers/gnoswap-service-provider/GnoswapServiceProvider.tsx

```diff
@@ -84,6 +68,34 @@ const GnoswapServiceProvider: React.FC<React.PropsWithChildren> = ({
     return new TokenRepositoryImpl(networkClient, localStorageClient);
   }, [localStorageClient, networkClient]);
 
+  async function initNetwork() {
+    const defaultChainId = process.env.NEXT_PUBLIC_DEFAULT_CHAIN_ID || "";
```

Comment:
I have an idea for process.env value. We can default this in constant and import this. To easy manage variable

Reply:
Are you suggesting to manage `DEFAULT_CHAIN_ID` as a constant variable?

Reply:
Yep.